### PR TITLE
RDM-10044 - New Publish Column for Case Events

### PR DESCRIPTION
**Pull request by smathangi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-10044
### Change description ###

New Publish Column for Case Events

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -261,7 +261,7 @@ subprojects { subproject ->
         testCompile "io.rest-assured:rest-assured:${restAssuredVersion}"
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
-        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.21.0'
+        testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.23.0-RDM-10044'
```
Comment (tomxelliott) [inline build.gradle:264]: Presumably this will be updated once the release is made final?

**Review by tomxelliott — APPROVED** [review]
Looks good to me. Just one minor comment on BEFTA version number

**Review by sullivangareth — APPROVED** [review]

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Comment by smathangi** [comment]
Closing as another one opened against latest master https://github.com/hmcts/ccd-definition-store-api/pull/849

**Pull request closed without merging**